src/app/ui/components/item/item.tsx

```diff
@@ -0,0 +1,45 @@
+import { forwardRef } from 'react';
+
+import { css } from 'leather-styles/css';
+import { styled } from 'leather-styles/jsx';
+import { type ItemVariantProps, item as itemRecipe } from 'leather-styles/recipes';
+import { HTMLStyledProps } from 'leather-styles/types';
+
+import { HasChildren } from '@app/common/has-children';
+
+export const itemCaptionStyles = css({
+  _groupDisabled: { color: 'accent.non-interactive' },
+  color: 'accent.text-subdued',
+});
+
+export const itemChevronStyles = css({
+  _groupDisabled: { color: 'accent.non-interactive' },
+  color: 'accent.action-primary-default',
+});
+
+const ItemBase = forwardRef<HTMLDivElement, HTMLStyledProps<'div'> & ItemVariantProps>(
+  (props, ref) => <styled.div className={`group ${itemRecipe()}`} ref={ref} {...props} />
+);
+
+const ItemInteractive = forwardRef<HTMLButtonElement, HTMLStyledProps<'button'> & ItemVariantProps>(
+  (props, ref) => (
+    <styled.button className={`group ${itemRecipe({ interactive: true })}`} ref={ref} {...props} />
```

Comment:
What's the group class about?

Reply:
`group` selectors in panda are pretty cool. This is how I'm handling the `isDisabled` state of children. You then apply `groupDisabled` to the children and they will inherit that state.

https://panda-css.com/docs/concepts/conditional-styles#group-selectors